### PR TITLE
Style Text Fields according guideline

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] #877
[Problem] Wrong Inpu Text styling. Wrong example
[Solution] Add new input text API for list. Style it properly. Modify example.
Place example in proper menu

### CODE REVIEW

File: examples/mobile/UIComponents/components/content/textinput.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	var page = document.getElementById("textinput-demo"),
+		textInputs = document.querySelectorAll(".input-hint");
+
+	page.addEventListener("pagebeforeshow", function () {
+		var idx,
+			onFocusBound,
+			onBlurBound;
+
+		onFocusBound = onFocus.bind(null);
+		onBlurBound = onBlur.bind(null);
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/content/textinput.js]: How about assigning the value with defining the value?

File: examples/mobile/UIComponents/components/textfields/textfieldswithoutlabels.html [hunk 2]
```diff
@@ -13,34 +13,35 @@
...
-				Text fields
+				Text fields without labels
 			</h1>
 		</header>
 		<!-- /header -->
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
-				<li class="ui-li-static">
-					<input placeholder="Hint text Inactive" type="text"/>
+				<li class="ui-li-text-input">
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/textfields/textfieldswithoutlabels.html]: It think "ui-li-has-text-input" is more proper to sync with other component such as 'ui-li-has-checkbox', or 'ui-li-has-icon'

File: examples/mobile/UIComponents/components/content/textinput.js (hunk 1 above)
Comment (grzegorz-czajkowski) [inline examples/mobile/UIComponents/components/content/textinput.js]: I think bind with null is not necessary? Can we pass onFocus/onBlur to addEventListener directly?

File: examples/mobile/UIComponents/components/content/textinput.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+		for (idx = 0; idx < textInputs.length; idx++) {
```
Comment (grzegorz-czajkowski) [inline examples/mobile/UIComponents/components/content/textinput.js]: How about using foreach ?

Comment (lslachciak) [reply examples/mobile/UIComponents/components/content/textinput.js]: done

File: examples/mobile/UIComponents/components/content/textinput.js (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/UIComponents/components/content/textinput.js]: Code simplified

File: examples/mobile/UIComponents/components/textfields/textfieldswithoutlabels.html (hunk 2 above)
Comment (lslachciak) [reply examples/mobile/UIComponents/components/textfields/textfieldswithoutlabels.html]: right, fixed

File: examples/mobile/UIComponents/components/content/textinput.js (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/UIComponents/components/content/textinput.js]: I simplified code now

**Review by grzegorz-czajkowski — APPROVED** [review]

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**